src/fdmdv2_main.cpp

```diff
@@ -1111,7 +1116,7 @@ void MainFrame::OnTimer(wxTimerEvent &evt)
 	m_textSync->SetLabel("Modem");
      }
     g_prev_State = g_State;
-    if (g_mode == FREEDV_MODE_700D) {
+    if ((g_mode == FREEDV_MODE_700D) || (g_mode == FREEDV_MODE_2020)){
```

Comment:
Good thinking! Hhowever the interleaver (at least past one frame) seems really used, so I doubt it will be used for 2020

Reply:
The interleaver might be good for broadcasting.